### PR TITLE
EZP-29599: Improved spacing among field types in Edit view

### PR DESCRIPTION
**Pull request by inakijv**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-29599](https://jira.ez.no/browse/EZP-29599)
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Most important aspects:
- Harmonized spacing among field types in Edit view, so that this view has a balanced UI structure making it easier to scan information for users;
- Added extra off-set margin-bottom.

ez platform 12
ez platform 13
ez platform 14

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/scss/fieldType/_ezformlabels.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.ez-content-edit {
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/fieldType/_ezformlabels.scss]: During our conversation I didn't know how the CSS selectors will look like in the final version. Now I see that all the rules that begin with `.ez-content-edit` should be placed in the `_content-edit.scss` file.
Please, move all the rules to the file I mentioned above :) The same applies for the rule you've added in the `_base-field.scss`

**Comment by inakijv** [comment]
Updated based on review.

**Comment by SylvainGuittard** [comment]
@inakijv ImageAsset has been merged (https://github.com/ezsystems/ezplatform-admin-ui/pull/580).
Maybe you should check if the spacing is correct on this new field.

**Comment by inakijv** [comment]
Thanks @SylvainGuittard. Tested, improvements are applied 👍 

ez platform 4
ez platform 5
ez platform 6

**Comment by barbaragr** [comment]
I've got some problems with displaying ezimageassets:
screen shot 2018-09-17 at 12 36 30

**Comment by inakijv** [comment]
@barbaragr I am trying to reproduce it, Could you provide more details, please?

**Comment by inakijv** [comment]
Updated my local instance and everything looks good.
Could you give more details?

ez platform 19
ez platform 18

**Comment by inakijv** [comment]
Rebased.

**Pull request merged**